pkg/kubemark/hollow_proxy.go

```diff
@@ -42,6 +45,24 @@ type HollowProxy struct {
 	ProxyServer *proxyapp.ProxyServer
 }
 
+type FakeProxyHandler struct{}
```

Comment:
Instead of this artificial type, let's make all the following methods methods of FakeProxier.

With that you should be able to avoid this strange casting below.